vyper/builtin_functions/functions.py

```diff
@@ -1302,6 +1302,22 @@ def build_IR(self, expr, args, kwargs, contact):
         )
 
 
+class RawRevert(BuiltinFunction):
+
+    _id = "raw_revert"
+    _inputs = [("data", BytesAbstractType())]
```

Comment:
i think this will typecheck bytesM (ex. bytes32) types but the codegen will not work

Reply:
Yeah, I can only successfully compile `Bytes[N]` currently. Do you think the type should be more restrictive, or should I look into implementing for bytes32 and others?

Reply:
the type should be more restrictive

Reply:
changed to `ArrayValueAbstractType`